app/src/utils/utils.ts

```diff
@@ -85,7 +85,7 @@ export function cleanTwitterUrl(url: string | undefined) {
 }
 
 // Given a metadata pointer object, return a stringified version that can be used as an object key
-export function metadataId(metaPtr: MetaPtr): string {
+export function metadataId(metaPtr: MetaPtr = { protocol: 0, pointer: '' }): string {
```

Comment:
What's the use case for setting this default, i.e. when would we call `metadataId()` to return a value of `0-`, without providing an input?

Reply:
This handles the case where metadata hasn't yet loaded, specifically on the cart.